docs/plugins/known-plugins.asciidoc

```diff
@@ -18,6 +18,7 @@ This list of plugins is not guaranteed to work on your version of Kibana. Instea
 * https://github.com/TrumanDu/indices_view[Indices View] - View indices related information.
 * https://github.com/johtani/analyze-api-ui-plugin[Analyze UI] (johtani) - UI for elasticsearch _analyze API
 * https://github.com/TrumanDu/cleaner[Cleaner] (TrumanDu)- Setting index ttl.
+* https://github.com/bitsensor/elastalert-kibana-plugin[ElastAlert Kibana Plugin] - UI to create, test and edit ElastAlert rules
```

Comment:
Hi, since this seems to require a fork of the "original" ElastAlert, could you please ass `bitsensor` as an author behind the link (as in the other links)?

Reply:
Resolved in 9fb15d5aaac5693a0024bf4f72490a396ef1ee1b